packages/field/src/FormRegistrarMixin.js

```diff
@@ -38,6 +38,7 @@ export const FormRegistrarMixin = dedupeMixin(
           super.connectedCallback();
         }
         formRegistrarManager.add(this);
+        formRegistrarManager.becomesReady(this);
```

Comment:
you will probably need to have a 
```js
if (this.__hasBeenRenedered) {
  formRegistrarManager.becomesReady(this);
}

firstUpdated(changedProperties) {
  // ...
  this.__hasBeenRenedered = true; // also pls init it with fall in the constructor
  // or is there a LitElement flag for that state already?
}
```

in there to make sure it does not become ready before it has been rendered at least once (to not break polyfilled browsers)

can you test it that would still work in your use case?

a test would be highly appreciated 🤗